### ISSUE TITLE
Colorpicker widget option to send color data at each highlight move

### ISSUE THREAD

**Issue by anordvall** [igorgladkov/rotini#27]
Hi!

This is a low prio feature request, but it would be useful (and cool in demos :) if the color picker could be configured to send the new color code to the server at each move of the highlight in the palette, and not waiting for the user to let go of the screen.

**Comment by fsedarkalex** [comment]
If you do this I would request to add that as an option (as suggested) not fixeed as some bindings might have problems with too high amounts of updates and it might cause lag

**Comment by d0t101101** [comment]
Totally agree here - quick interface updates are preferred (I would use it!) but it has the potential to bog down the targets - so make it configurable.  

@anordvall - your described approach works well in the native OpenHAB Web UI with color widgets, but not so sure this would be as reliable or real-time on low power setups if many changes are made in quick succession...

**Comment by fsedarkalex** [comment]
Well I am using DMX binding and it lasts about 3 seconds to update all
15 used channels imagine it would have to do more than one update within
that time. 

Or imagine you are using rules to switch on/off a second light at
specific brightness values of your rgb and you are using RFXCOM binding.
the RfxTrx needs about 1-2 seconds to transmit a message. Same problem
here. 

That's why I suggest that to be configurable (probably even on a
per-widget base)

**Comment by anordvall** [comment]
Yes, a per-widget configuration for this is what I was hoping for. Of course the lag will depend on several things, for instance how fast you move your finger :smile:
